--- configure.ac.orig	2006-01-21 12:34:02.000000000 +0100
+++ configure.ac	2006-01-21 12:34:41.000000000 +0100
@@ -825,7 +825,7 @@
 AC_DEFINE_UNQUOTED(SYSCONFDIR, "$_sysconfdir", [where the configuration is stored (defined by configure)])
 
 
-dnl Support for GNUSTEP_LOCAL_ROOT, for WPrefs.app
+dnl Support for GNUSTEP_ROOT, for WPrefs.app
 dnl ==============================================
 
 AC_ARG_WITH(appspath,
@@ -833,15 +833,15 @@
 
 appspath=$with_gnustepdir
 
-if test "x$appspath$GNUSTEP_LOCAL_ROOT" = "x"; then
+if test "x$appspath$GNUSTEP_ROOT" = "x"; then
     wprefs_base_dir=${prefix}
     wprefs_datadir="${datadir}/WPrefs"
     wprefs_bindir="${bindir}"
 else
     gnustepdir=$appspath
 
-    if test "x$GNUSTEP_LOCAL_ROOT" != "x" ; then
-	gnustepdir=`echo "$GNUSTEP_LOCAL_ROOT" | sed -e "s|^${prefix}|prefix|"`
+    if test "x$GNUSTEP_ROOT" != "x" ; then
+	gnustepdir=`echo "$GNUSTEP_ROOT" | sed -e "s|^${prefix}|prefix|"`
 	gnustepdir=`echo $gnustepdir | sed -e 's|^prefix|${prefix}|'`
     fi
 
