diff -urN WindowMaker-0.92.0.orig/src/Makefile.am WindowMaker-0.92.0/src/Makefile.am
--- WindowMaker-0.92.0.orig/src/Makefile.am	2005-06-21 19:48:05.759714744 +0200
+++ WindowMaker-0.92.0/src/Makefile.am	2005-06-22 09:15:55.716050968 +0200
@@ -117,7 +117,7 @@
 
 
 wmaker_LDADD = \
-	$(top_builddir)/WINGs/libWINGs.a\
+	$(top_builddir)/WINGs/libWINGs.la\
 	$(top_builddir)/wrlib/libwraster.la\
 	@XFTLIBS@ \
 	@XLIBS@ \
diff -urN WindowMaker-0.92.0.orig/test/Makefile.am WindowMaker-0.92.0/test/Makefile.am
--- WindowMaker-0.92.0.orig/test/Makefile.am	2005-06-21 19:48:05.978681456 +0200
+++ WindowMaker-0.92.0/test/Makefile.am	2005-06-22 09:17:55.086903840 +0200
@@ -8,7 +8,7 @@
 
 wtest_SOURCES = wtest.c
 
-wtest_LDADD = $(top_builddir)/wmlib/libWMaker.a @XLFLAGS@ @XLIBS@ 
+wtest_LDADD = $(top_builddir)/wmlib/libWMaker.la @XLFLAGS@ @XLIBS@ 
 
 
 INCLUDES = -g -D_BSD_SOURCE @XCFLAGS@ -I$(top_srcdir)/wmlib 
diff -urN WindowMaker-0.92.0.orig/util/Makefile.am WindowMaker-0.92.0/util/Makefile.am
--- WindowMaker-0.92.0.orig/util/Makefile.am	2005-06-21 19:48:05.787710488 +0200
+++ WindowMaker-0.92.0/util/Makefile.am	2005-06-22 09:24:04.266779968 +0200
@@ -1,8 +1,6 @@
 
 AUTOMAKE_OPTIONS = no-dependencies
 
-pkgdatadir  = $(datadir)/@PACKAGE@
-
 bin_PROGRAMS = wxcopy wxpaste wdwrite wdread getstyle setstyle convertfonts \
 	seticons geticonset wmsetbg wmsetup wmagnify
 
@@ -11,49 +9,48 @@
 EXTRA_DIST = wmaker.inst.in bughint wm-oldmenu2new directjpeg.c wkdemenu.pl
 
 INCLUDES = $(DFLAGS) -I$(top_srcdir)/WINGs -I$(top_srcdir)/wrlib \
-	@HEADER_SEARCH_PATH@ \
-	-DETCDIR=\"sysconfdir\" -DDATADIR=\"pkgdatadir\"
-
-liblist= @LIBRARY_SEARCH_PATH@ @INTLIBS@
+       $(HEADER_SEARCH_PATH) \
+       -DETCDIR=\"$(sysconfdir)\" -DDATADIR=\"$(pkgdatadir)\"
 
-wdwrite_LDADD = $(top_builddir)/WINGs/libWUtil.a $(liblist)
+liblist= $(LIBRARY_SEARCH_PATH) $(INTLIBS)
+wdwrite_LDADD = $(top_builddir)/WINGs/libWUtil.la $(liblist)
 
-wdread_LDADD = $(top_builddir)/WINGs/libWUtil.a $(liblist)
+wdread_LDADD = $(top_builddir)/WINGs/libWUtil.la $(liblist)
 
-wxcopy_LDADD = @XLFLAGS@ @XLIBS@ 
+wxcopy_LDADD = $(XLFLAGS) $(XLIBS)
 
-wxpaste_LDADD = @XLFLAGS@ @XLIBS@
+wxpaste_LDADD = $(XLFLAGS) $(XLIBS)
 
-getstyle_LDADD = $(top_builddir)/WINGs/libWUtil.a $(liblist)
+getstyle_LDADD = $(top_builddir)/WINGs/libWUtil.la $(liblist)
 
 getstyle_SOURCES = getstyle.c fontconv.c
 
 setstyle_LDADD = \
-	$(top_builddir)/WINGs/libWUtil.a \
-	@XLFLAGS@ @XLIBS@ $(liblist)
+	$(top_builddir)/WINGs/libWUtil.la \
+	$(XLFLAGS) $(XLIBS) $(liblist)
 
 setstyle_SOURCES = setstyle.c fontconv.c
 
-convertfonts_LDADD = $(top_builddir)/WINGs/libWUtil.a $(liblist)
+convertfonts_LDADD = $(top_builddir)/WINGs/libWUtil.la $(liblist)
 
 convertfonts_SOURCES = convertfonts.c fontconv.c
 
-seticons_LDADD= $(top_builddir)/WINGs/libWUtil.a $(liblist)
+seticons_LDADD= $(top_builddir)/WINGs/libWUtil.la $(liblist)
 
-geticonset_LDADD= $(top_builddir)/WINGs/libWUtil.a $(liblist)
+geticonset_LDADD= $(top_builddir)/WINGs/libWUtil.la $(liblist)
 
 wmagnify_LDADD = \
-	$(top_builddir)/WINGs/libWINGs.a \
+	$(top_builddir)/WINGs/libWINGs.la \
 	$(top_builddir)/wrlib/libwraster.la \
 	@XFTLIBS@ @INTLIBS@ @DLLIBS@
 
 wmsetup_LDADD = \
-	$(top_builddir)/WINGs/libWINGs.a \
+	$(top_builddir)/WINGs/libWINGs.la \
 	$(top_builddir)/wrlib/libwraster.la \
 	@XFTLIBS@ @INTLIBS@ @DLLIBS@
 
 wmsetbg_LDADD = \
-	$(top_builddir)/WINGs/libWINGs.a \
+	$(top_builddir)/WINGs/libWINGs.la \
 	$(top_builddir)/wrlib/libwraster.la \
 	@XLIBS@ @XFTLIBS@ @INTLIBS@ @DLLIBS@
 
diff -urN WindowMaker-0.92.0.orig/WINGs/Examples/Makefile.am WindowMaker-0.92.0/WINGs/Examples/Makefile.am
--- WindowMaker-0.92.0.orig/WINGs/Examples/Makefile.am	2005-06-21 19:48:05.731719000 +0200
+++ WindowMaker-0.92.0/WINGs/Examples/Makefile.am	2005-06-22 08:51:20.892258264 +0200
@@ -6,27 +6,14 @@
 noinst_PROGRAMS = connect server fontl puzzle colorpick
 
 
-LDADD= $(top_builddir)/WINGs/libWINGs.a $(top_builddir)/wrlib/libwraster.la \
-	@XFTLIBS@ @INTLIBS@
-
-colorpick_DEPENDENCIES = $(top_builddir)/WINGs/libWINGs.a
-
-fontl_DEPENDENCIES = $(top_builddir)/WINGs/libWINGs.a
-
-puzzle_DEPENDENCIES = $(top_builddir)/WINGs/libWINGs.a
-
-connect_DEPENDENCIES = $(top_builddir)/WINGs/libWUtil.a
-
-connect_LDADD = $(top_builddir)/WINGs/libWUtil.a @LIBRARY_SEARCH_PATH@ \
-	@NETLIBS@ @INTLIBS@
-
-server_DEPENDENCIES = $(top_builddir)/WINGs/libWUtil.a
-
-server_LDADD = $(top_builddir)/WINGs/libWUtil.a @LIBRARY_SEARCH_PATH@ \
-	@NETLIBS@ @INTLIBS@
+LDADD= $(top_builddir)/WINGs/libWINGs.la $(top_builddir)/wrlib/libwraster.la \
+       $(XFTLIBS) $(INTLIBS)
+connect_LDADD = $(top_builddir)/WINGs/libWUtil.la $(LIBRARY_SEARCH_PATH) \
+       $(NETLIBS) $(INTLIBS)
 
+server_LDADD = $(top_builddir)/WINGs/libWUtil.la $(LIBRARY_SEARCH_PATH) \
+       $(NETLIBS) $(INTLIBS)
 
 INCLUDES = -I$(top_srcdir)/WINGs -I$(top_srcdir)/wrlib -I$(top_srcdir)/src \
-	-DRESOURCE_PATH=\"$(datadir)/WINGs\" @HEADER_SEARCH_PATH@ -DDEBUG
-
+	-DRESOURCE_PATH=\"$(datadir)/WINGs\" $(HEADER_SEARCH_PATH) -DDEBUG
 
diff -urN WindowMaker-0.92.0.orig/WINGs/Extras/Makefile.am WindowMaker-0.92.0/WINGs/Extras/Makefile.am
--- WindowMaker-0.92.0.orig/WINGs/Extras/Makefile.am	2005-06-21 19:48:05.732718848 +0200
+++ WindowMaker-0.92.0/WINGs/Extras/Makefile.am	2005-06-22 08:55:37.785204600 +0200
@@ -2,21 +2,16 @@
 
 AUTOMAKE_OPTIONS = no-dependencies
 
-
-
 # is this a kluge? if so, how should i do it?
 includedir = @includedir@/WINGs
 
 include_HEADERS = wtableview.h wtabledelegates.h 
 
-
-lib_LIBRARIES = libExtraWINGs.a 
-
+lib_LTLIBRARIES = libExtraWINGs.la
 noinst_PROGRAMS = test
 
-EXTRA_DIST = 
 
-libExtraWINGs_a_SOURCES = 	\
+libExtraWINGs_la_SOURCES =     \
 	wtableview.c \
 	wtabledelegates.c \
 	wtableview.h \
@@ -25,10 +20,9 @@
 AM_CFLAGS = @NOSTRICTALIASING@
 
 INCLUDES = -I$(top_srcdir)/wrlib -I$(top_srcdir)/WINGs \
-	-DRESOURCE_PATH=\"$(datadir)/WINGs\" @HEADER_SEARCH_PATH@ -DDEBUG
-
-LDADD= $(top_builddir)/WINGs/libWINGs.a $(top_builddir)/wrlib/libwraster.la \
-	@XFTLIBS@ @INTLIBS@
+	-DRESOURCE_PATH=\"$(datadir)/WINGs\" $(HEADER_SEARCH_PATH)
+LDADD= $(top_builddir)/WINGs/libWINGs.la $(top_builddir)/wrlib/libwraster.la \
+       $(XFTLIBS) $(INTLIBS)
 
 test_LDADD = wtableview.o wtabledelegates.o $(LDADD)
 
diff -urN WindowMaker-0.92.0.orig/WINGs/Makefile.am WindowMaker-0.92.0/WINGs/Makefile.am
--- WindowMaker-0.92.0.orig/WINGs/Makefile.am	2005-06-21 19:48:05.651731160 +0200
+++ WindowMaker-0.92.0/WINGs/Makefile.am	2005-06-22 09:10:48.046823800 +0200
@@ -4,16 +4,12 @@
 
 SUBDIRS = WINGs . po Documentation Resources Examples Extras Tests
 
-
-
-#libWINGs_la_LDFLAGS = -version-info 1:1:0
-
 bin_SCRIPTS = get-wings-flags get-wutil-flags
 
-lib_LIBRARIES = libWINGs.a libWUtil.a
+LDADD= libWINGs.a $(top_builddir)/wrlib/libwraster.la @INTLIBS@
 
+lib_LTLIBRARIES = libWINGs.la libWUtil.la
 
-LDADD= libWINGs.a $(top_builddir)/wrlib/libwraster.la @INTLIBS@
 
 
 EXTRA_DIST = BUGS python/Makefile python/README python/WINGs.i \
@@ -21,7 +17,7 @@
 
 
 # wbutton.c 
-libWINGs_a_SOURCES = 	\
+libWINGs_la_SOURCES = 	\
 	array.c \
 	bagtree.c \
 	configuration.c \
@@ -83,7 +79,7 @@
 	wwindow.c
 
 
-libWUtil_a_SOURCES = 	\
+libWUtil_la_SOURCES = 	\
 	array.c \
 	bagtree.c \
 	connection.c \
diff -urN WindowMaker-0.92.0.orig/WINGs/Tests/Makefile.am WindowMaker-0.92.0/WINGs/Tests/Makefile.am
--- WindowMaker-0.92.0.orig/WINGs/Tests/Makefile.am	2005-06-21 19:48:05.733718696 +0200
+++ WindowMaker-0.92.0/WINGs/Tests/Makefile.am	2005-06-22 09:13:09.083382984 +0200
@@ -4,19 +4,14 @@
 
 noinst_PROGRAMS = wtest wmquery wmfile testmywidget 
 
-LDADD= $(top_builddir)/WINGs/libWINGs.a $(top_builddir)/wrlib/libwraster.la \
-	@XFTLIBS@ @INTLIBS@
-
+LDADD= $(top_builddir)/WINGs/libWINGs.la $(top_builddir)/wrlib/libwraster.la \
+       $(XFTLIBS) $(INTLIBS)
 
 testmywidget_SOURCES = testmywidget.c mywidget.c mywidget.h 
 
-wtest_DEPENDENCIES = $(top_builddir)/WINGs/libWINGs.a
-
 
 EXTRA_DIST = logo.xpm upbtn.xpm wm.html wm.png
 
 INCLUDES = -I$(top_srcdir)/WINGs -I$(top_srcdir)/wrlib -I$(top_srcdir)/src \
-	-DRESOURCE_PATH=\"$(datadir)/WINGs\" @XFTFLAGS@ @HEADER_SEARCH_PATH@ \
-	-DDEBUG
-
+	 -DRESOURCE_PATH=\"$(datadir)/WINGs\" $(HEADER_SEARCH_PATH)
 
diff -urN WindowMaker-0.92.0.orig/wmlib/Makefile.am WindowMaker-0.92.0/wmlib/Makefile.am
--- WindowMaker-0.92.0.orig/wmlib/Makefile.am	2005-06-21 19:48:05.977681608 +0200
+++ WindowMaker-0.92.0/wmlib/Makefile.am	2005-06-22 09:25:59.649239168 +0200
@@ -1,13 +1,13 @@
 
 AUTOMAKE_OPTIONS = no-dependencies
 
-lib_LIBRARIES = libWMaker.a
+lib_LTLIBRARIES = libWMaker.la
 
 include_HEADERS = WMaker.h
 
 INCLUDES = $(DFLAGS) @XCFLAGS@
 
-libWMaker_a_SOURCES = 	\
+libWMaker_la_SOURCES = 	\
 	menu.c 		\
 	app.c 		\
 	event.c 	\
@@ -15,6 +15,9 @@
 	app.h		\
 	menu.h	
 
+libWMaker_la_LDFLAGS = -version-info 1:1:1
+
+	
 DISTCLEANFILES = wmlib.pc
 
 install-exec-local:
diff -urN WindowMaker-0.92.0.orig/WPrefs.app/Makefile.am WindowMaker-0.92.0/WPrefs.app/Makefile.am
--- WindowMaker-0.92.0.orig/WPrefs.app/Makefile.am	2005-06-21 19:48:05.979681304 +0200
+++ WindowMaker-0.92.0/WPrefs.app/Makefile.am	2005-06-22 09:15:21.966181728 +0200
@@ -48,10 +48,10 @@
 
 INCLUDES = -I$(top_srcdir)/wrlib -I$(top_srcdir)/WINGs @HEADER_SEARCH_PATH@ 
 
-WPrefs_DEPENDENCIES = $(top_builddir)/WINGs/libWINGs.a
+WPrefs_DEPENDENCIES = $(top_builddir)/WINGs/libWINGs.la
 
 WPrefs_LDADD = \
-	$(top_builddir)/WINGs/libWINGs.a\
+	$(top_builddir)/WINGs/libWINGs.la\
 	$(top_builddir)/wrlib/libwraster.la \
 	@XFTLIBS@ \
 	@INTLIBS@
