--- WindowMaker-0.51.2/WindowMaker/Defaults/WindowMaker.in.old	Tue Feb 16 20:21:45 1999
+++ WindowMaker-0.51.2/WindowMaker/Defaults/WindowMaker.in	Fri Mar 12 22:56:38 1999
@@ -9,7 +9,8 @@
     "/usr/X11R6/include/X11/pixmaps",
     "~/GNUstep/Library/WindowMaker/Pixmaps",
     "/usr/local/share/pixmaps",
-    "/usr/share/pixmaps"
+    "/usr/share/pixmaps",
+    "/usr/X11R6/share/pixmaps"
   );
   PixmapPath = (
     "~/GNUstep/Library/WindowMaker/Pixmaps",
@@ -18,7 +19,8 @@
     "#pkgdatadir#/Backgrounds",
     "/usr/X11R6/include/X11/pixmaps",
     "/usr/local/share/pixmaps",
-    "/usr/share/pixmaps"
+    "/usr/share/pixmaps",
+    "/usr/X11R6/share/pixmaps"
   );
   WindowTitleBalloons = YES;
   IconSize = 64;
