diff -urN WindowMaker-0.50.2.old/WindowMaker/menu WindowMaker-0.50.2/WindowMaker/menu
--- WindowMaker-0.50.2.old/WindowMaker/menu	Tue Jan 12 13:48:53 1999
+++ WindowMaker-0.50.2/WindowMaker/menu	Mon Jan 18 16:36:38 1999
@@ -111,48 +109,10 @@
 		"Process List" EXEC xterm -e top
 		"Manual Browser" EXEC xman
 	"Info" END
-	"Run..." EXEC %a(Run,Type command to run:)
 	"XTerm" EXEC xterm -sb 
 	"Rxvt" EXEC rxvt -bg black -fg white -fn fixed
 	"Workspaces" WORKSPACE_MENU
-	"Applications" MENU
-		"Graphics" MENU
-			"Gimp" EXEC gimp >/dev/null
-			"XV" EXEC xv
-			"XPaint" EXEC xpaint
-			"XFig" EXEC xfig
-		"Graphics" END
-		"X File Manager" EXEC xfm
-		"OffiX Files" EXEC files
-		"LyX" EXEC lyx
-		"Netscape" EXEC netscape 
-  		"Ghostview" EXEC ghostview %a(GhostView,Enter file to view)
-		"Acrobat" EXEC /usr/local/Acrobat3/bin/acroread %a(Acrobar,Enter PDF to view)
-  		"TkDesk" EXEC tkdesk
-	"Applications" END
-	"Editors" MENU
-		"XFte" EXEC xfte
-		"XEmacs" EXEC xemacs || emacs
-		"XJed" EXEC xjed 
-		"NEdit" EXEC nedit
-		"Xedit" EXEC xedit
-		"VI" EXEC xterm -e vi
-	"Editors" END
-	"Miscellaneous" MENU
-		"Xmcd" EXEC xmcd 2> /dev/null
-		"Xplaycd" EXEC xplaycd
-		"Xmixer" EXEC xmixer
-	"Miscellaneous" END
-	"Utils" MENU
-		"Calculator" EXEC xcalc
-		"Window Properties" EXEC xprop | xmessage -center -title 'xprop' -file -
-		"Font Chooser" EXEC xfontsel
-		"Terminal Emulator" EXEC xminicom
-		"Magnify" EXEC xmag
-		"Colormap" EXEC xcmap
-		"XKill" EXEC xkill
-		"Clipboard" EXEC xclipboard
-	"Utils" END
+	"Applications" OPEN_MENU | wmconfig --output wmaker --rootmenu Applications 2>/dev/null 
 
 	"Selection" MENU
 		"Copy" EXEC echo '%s' | wxcopy
diff -urN WindowMaker-0.50.2.old/WindowMaker/plmenu WindowMaker-0.50.2/WindowMaker/plmenu
--- WindowMaker-0.50.2.old/WindowMaker/plmenu	Tue Jan 12 13:48:53 1999
+++ WindowMaker-0.50.2/WindowMaker/plmenu	Mon Jan 18 19:51:05 1999
@@ -11,45 +11,7 @@
  ("XTerm", EXEC, "xterm -sb"),
  ("Rxvt", EXEC, "rxvt -bg black -fg white -fn fixed"),
  ("Workspaces", WORKSPACE_MENU),
- ("Applications",
-  ("Graphics",
-   ("Gimp", EXEC, "gimp >/dev/null"),
-   ("XV", EXEC, "xv"),
-   ("XPaint", EXEC, "xpaint"),
-   ("XFig", EXEC, "xfig")
-   ),
-  ("X File Manager", EXEC, "xfm"),
-  ("OffiX Files", EXEC, "files"),
-  ("LyX", EXEC, "lyx"),
-  ("Netscape", EXEC, "netscape"),
-  ("Ghostview", EXEC, "ghostview %a(Enter file to view)"),
-  ("Acrobat", EXEC, "/usr/local/Acrobat3/bin/acroread %a(Enter PDF to view)"),
-  ("TkDesk", EXEC, "tkdesk")
-  ),
- ("Editors",
-  ("XFte", EXEC, "xfte"),
-  ("XEmacs", EXEC, "xemacs || emacs"),
-  ("XJed", EXEC, "xjed"),
-  ("NEdit", EXEC, "nedit"),
-  ("Xedit", EXEC, "xedit"),
-  ("VI", EXEC, "xterm -e vi")
-  ),
- ("Miscellaneous",
-  ("Xmcd", EXEC, "xmcd 2> /dev/null"),
-  ("Xplaycd", EXEC, "xplaycd"),
-  ("Xmixer", EXEC, "xmixer")
-  ),
- ("Utils",
-  ("Calculator", EXEC, "xcalc"),
-  ("Window Properties", EXEC, "xprop | xmessage -center -title 'xprop' -file -"),
-  ("Font Chooser", EXEC, "xfontsel"),
-  ("Terminal Emulator", EXEC, "xminicom"),
-  ("Magnify", EXEC, "xmag"),
-  ("Colormap", EXEC, "xcmap"),
-  ("XKill", EXEC, "xkill"),
-  ("ASClock", EXEC, "asclock -shape"),
-  ("Clipboard", EXEC, "xclipboard")
-  ),
+ ("Applications", OPEN_MENU, "| wmconfig --output wmaker --rootmenu Applications 2>/dev/null "),
  ("Selection",
   ("Copy", EXEC, "echo '%s' | wxcopy"),
   ("Mail To", EXEC, "xterm -name mail -T Pine -e pine %s"),
