--- WindowMaker-0.52.0/src/main.c.runinst	Tue Mar 16 06:26:29 1999
+++ WindowMaker-0.52.0/src/main.c	Mon Apr 19 15:29:24 1999
@@ -277,11 +277,8 @@
     char *path;
 
     path = wdefaultspathfordomain("");
-    if (access(path, R_OK)!=0) {
-	wfatal(_("could not find user GNUstep directory (%s).\n"
-		 "Make sure you have installed Window Maker correctly and run wmaker.inst"),
-	       path);
-	exit(1);
+    if (access(path, R_OK)!=0) {	
+      system("wmaker.inst --batch");
     }
     
     free(path);
