diff -rU 2 WindowMaker-0.62.1/WindowMaker/Defaults/WindowMaker.in WindowMaker-0.62.1.tlh/WindowMaker/Defaults/WindowMaker.in
--- WindowMaker-0.62.1/WindowMaker/Defaults/WindowMaker.in	Tue Sep 21 22:09:43 1999
+++ WindowMaker-0.62.1.tlh/WindowMaker/Defaults/WindowMaker.in	Wed Jul 19 14:19:16 2000
@@ -39,5 +39,5 @@
   NewStyle = YES;
   DisableDock = NO;
-  DisableClip = NO;
+  DisableClip = YES;
   Superfluous = NO;
   StickyIcons = NO;
@@ -52,5 +52,5 @@
   DisableAnimations = NO;
   DontLinkWorkspaces = YES;
-  AutoArrangeIcons = NO;
+  AutoArrangeIcons = YES;
   AdvanceToNewWorkspace = NO;
   CycleWorkspaces = NO;
@@ -58,5 +58,5 @@
   MoveDisplay = floating;
   OpaqueMove = YES;
-  IconPosition = "blh";
+  IconPosition = "trh";
   WrapMenus = NO;
   ScrollableMenus = YES;
