diff -ur WindowMaker-CVS-20040723.orig/WPrefs.app/Expert.c WindowMaker-CVS-20040723/WPrefs.app/Expert.c
--- WindowMaker-CVS-20040723.orig/WPrefs.app/Expert.c	2004-07-23 07:23:15.000000000 +0200
+++ WindowMaker-CVS-20040723/WPrefs.app/Expert.c	2004-09-05 04:26:07.050661872 +0200
@@ -84,7 +84,7 @@
     WMSetButtonText(panel->swi[7], _("Smooth font edges (needs restart)."));
 
     //if (!WMHasAntialiasingSupport(WMWidgetScreen(panel->box)))
-        WMSetButtonEnabled(panel->swi[7], False);
+        //WMSetButtonEnabled(panel->swi[7], False);
 
     WMRealizeWidget(panel->box);
     WMMapSubwidgets(panel->box);
