diff -Nur WindowMaker-0.70.0.orig/src/misc.c WindowMaker-0.70.0/src/misc.c
--- WindowMaker-0.70.0.orig/src/misc.c	Thu Oct  4 05:39:18 2001
+++ WindowMaker-0.70.0/src/misc.c	Thu Oct 18 16:26:04 2001
@@ -679,7 +679,7 @@
 	 case _PROMPT:
 	    if (line[*ptr]==')' && j==0) {
 
-		if (*ptr-begin > 1) {
+		if (*ptr > begin) {
 		    strncpy(pbuffer, &line[begin], WMIN(*ptr-begin, BUFSIZE));
 		    pbuffer[WMIN(*ptr-begin, BUFSIZE)] = 0;
 		    prompt = (char*)pbuffer;
@@ -698,6 +698,7 @@
 #undef _PROMPT
 #undef _DONE
 
+    (*ptr)--; /* To be incremented in ExpandOptions */
     if (!wInputDialog(scr, title, prompt, &ret))
 	return NULL;
     else
