--- WindowMaker-CVS-20040321/WindowMaker/Defaults/WMWindowAttributes.in~	2004-02-08 06:01:02.000000000 +0100
+++ WindowMaker-CVS-20040321/WindowMaker/Defaults/WMWindowAttributes.in	2004-04-19 14:17:35.922181752 +0200
@@ -7,8 +7,10 @@
   WSoundServer = {Icon = sound.#extension#;};
   panel.Panel = {NoAppIcon = YES;};
   gmc.Gmc = {NoAppIcon = YES;};
-  XTerm = {Icon = Terminal.#extension#;};
-  NXTerm = {Icon = Terminal.#extension#;};
+  XTerm = {Icon = Terminal.#extension#;NoAppIcon = YES;};
+  NXTerm = {Icon = Terminal.#extension#;NoAppIcon = YES;};
+  rxvt = {Icon = Terminal.#extension#;NoAppIcon = YES;};
+  aterm = {Icon = Terminal.#extension#;NoAppIcon = YES;};
   ScilabGraphic0.Xscilab = {KeepInsideScreen=YES;};
   ScilabGraphic1.Xscilab = {KeepInsideScreen=YES;};
   ScilabGraphic2.Xscilab = {KeepInsideScreen=YES;};
