--- WindowMaker-0.80.1.orig/WindowMaker/menu.pl	Tue Jan  4 22:45:57 2000
+++ WindowMaker-0.80.1/WindowMaker/menu.pl	Fri Aug  2 10:28:45 2002
@@ -1,13 +1,13 @@
 /*
- * Definiowanie Menu Głównego dla WindowMakera
- * Fonty w standardzie ISO8895-2
+ * Definiowanie Menu Głównego WindowMakera
+ * Fonty w standardzie ISO8859-2
  *
  * Składnia jest następująca:
  *
- * <Tytuł> [SHORTCUT <Skrut>] <Komenda> <Paramery>
+ * <Tytuł> [SHORTCUT <Skrót>] <Komenda> <Parametry>
  *
  * <Tytuł> Tytuł może być dowolnym ciągiem znaków. Jeśli będą w nim występować
- *         spacje umieśc go w cudzysłowie np. "Tytuł ze spacją"
+ *         spacje umieść go w cudzysłowie np. "Tytuł ze spacją"
  * 
  * SHORTCUT Definiowanie skrótu.
  * <Skrót> Nazwa rodzaju skrótu umieszczonego w pliku
@@ -46,23 +46,22 @@
  *                  żadnych zmian w pliku SaveSessionOnExit .
  *  INFO - Wyświetlenie informacji o WindowMakerze
  *
- * <Parametry> zalezne od uruchamianego programu.
+ * <Parametry> zależne od uruchamianego programu.
  *
  * ** Opcje w lini komend EXEC:
- * %s - znak jest zastepowany przez text znajdujacy sie w ,,schowku''
- * %a(tytuł[,komunikat]) - otwiera dodatkowe okno o tytule tytuł, komunikacie
- *                         komunikat i czeka na podanie parametrów, które 
- *                         zostaną wstawione zamiast %a. Niestety nie udalo mi
- *                         się uzyskać polskich fontów w tej pocji :( 
- * %w - znak jest zastepowany przez XID aktywnego okna
- * %W - znak jest zastepowany przez numer aktywnego pulpitu
+ * %s - znak jest zastępowany przez text znajdujący się w 'schowku'
+ * %a(tytuł[,komunikat]) - otwiera dodatkowe okno o tytule 'tytuł', komunikacie
+ *                         'komunikat' i czeka na podanie parametrów, które 
+ *                         zostaną wstawione zamiast %a. 
+ * %w - znak jest zastępowany przez XID aktywnego okna
+ * %W - znak jest zastępowany przez numer aktywnego pulpitu
  * 
- * Aby używać specjalnych znaków ( takich jak % czy " ) należy poprzedzic je znakiem \
+ * Aby używać specjalnych znaków ( takich jak % czy " ) należy poprzedzić je znakiem \
  * np. :xterm -T "\"Witaj Świecie\""
  *
- * Można używac znaków specjalnych, takich jak \n
+ * Można używać znaków specjalnych, takich jak \n
  *
- * Sekcja MENU musi być zakończona sekcja END, pod tą sama nazwą.
+ * Sekcja MENU musi być zakończona sekcją END, pod tą samą nazwą.
  *
  * Przykład:
  *
@@ -70,7 +69,7 @@
  *  "XTerm" EXEC xterm
  *      // stworzenie podmenu z plikami w podkatalogu /usr/openwin/bin
  *  "XView apps" OPEN_MENU "/usr/openwin/bin"
- *      // umieszcza w jednym podmenu pliki z róznych podkatalogów
+ *      // umieszcza w jednym podmenu pliki z różnych podkatalogów
  *  "X11 apps" OPEN_MENU /usr/X11/bin $HOME/bin/X11
  *      // ustawienie tła
  *  "Background" OPEN_MENU -noext $HOME/images /usr/share/images WITH wmsetbg -u *      // wstawienie menu z pliku style.menu
@@ -95,142 +94,38 @@
                   
 
 #include "wmmacros"
-#define ULUB_EDYTOR vi 
-/* Jeśli nie lubisz edytora vi zmień na swój ulubiony edytor */
-#define ULUB_TERM xterm
-/* A tutaj ustaw swój ulubiony terminal */
 
 "WindowMaker" MENU
 	"Informacja" MENU
-		"Informacja o WMaker..." INFO_PANEL
-		"Legalność..."           LEGAL_PANEL
-		"Konsola Systemu"        EXEC xconsole
-		"Obciążenie Systemu"     EXEC xosview || xload
-		"Lista Procesów"         EXEC ULUB_TERM -T "Lista Procesów" -e top
-		"Przeglądarka Manuali"   EXEC xman
+		"Informacja o WindowMakerze..."      INFO_PANEL
+		"Informacja o licencji..." LEGAL_PANEL
+		"Konsola systemu"        EXEC xconsole
+		"Obciążenie systemu"     SHEXEC xosview || xload
+		"Lista procesów"         EXEC xterm -T "Lista procesów" -e top
+		"Przeglądarka manuali"   EXEC xman
 	"Informacja" END
 	
-	"Konfiguracja" MENU	
-		"Edycja menu"       EXEC ULUB_TERM -T "Edycja menu" -e ULUB_EDYTOR $HOME/GNUstep/Library/WindowMaker/menu
-		"Ustawienie fontów" EXEC ULUB_TERM -T "Ustawienie fontów" -e ULUB_EDYTOR $HOME/GNUstep/Defaults/WMGLOBAL
-		"Konfiguracja"      EXEC ULUB_TERM -T "Konfiguracja" -e ULUB_EDYTOR $HOME/GNUstep/Defaults/WindowMaker
-	"Konfiguracja" END
-	
-	"Uruchom..." EXEC %a(Uruchom,Wpisz komende do uruchomienia:)
-	"Terminal"   EXEC ULUB_TERM -T "Mój ulubiony terminal" -sb 
-	"Edytor"     EXEC ULUB_TERM -T "Moj ulubiony edytor" -e ULUB_EDYTOR %a(Edytor,Podaj plik do edycji:)
-	"Pulpity"    WORKSPACE_MENU
-	
-	"Aplikacje" MENU
-		"Grafika" MENU
-			"Gimp"        EXEC gimp >/dev/null
-			"XV"          EXEC xv
-			"XFig"        EXEC xfig
-			"XPaint"      EXEC xpaint
-			"Gnuplot"     EXEC ULUB_TERM -T "GNU plot" -e gnuplot
-			"Edytor ikon" EXEC bitmap
-		"Grafika" END
-		"Tekst" MENU
-			"LyX"                 EXEC lyx
-  			"Ghostview"           EXEC gv %a(GhostView,Wprowadz nazwe pliku *.ps *.pdf *.no:)
-  			"XDvi"                EXEC xdvi %a(XDvi,Wprowadz nazwe pliku *.dvi:)
-			"Acrobat"             EXEC /usr/local/Acrobat3/bin/acroread %a(Acrobat,Wprowadz nazwe pliku *.pdf:)
-			"Xpdf"                EXEC xpdf %a(Xpdf,Wprowadz nazwe pliku *.pdf:)
-			"Arkusz kalkulacyjny" EXEC xspread
-		"Tekst" END
-		"X File Manager"     EXEC xfm
-		"OffiX Files"        EXEC files
-		"TkDesk"             EXEC tkdesk
-		"Midnight Commander" EXEC ULUB_TERM -T "Midnight Commander" -e mc
-		"X Gnu debbuger"     EXEC xxgdb
-		"Xwpe"               EXEC xwpe
-	"Aplikacje" END
-	
-	"Internet" MENU
-		"Przeglądarki" MENU
-			"Netscape" EXEC netscape 
-			"Arena"    EXEC arena
-			"Lynx"     EXEC ULUB_TERM -e lynx %a(Lynx,Podaj URL:)
-		"Przeglądarki" END
-		"Programy pocztowe" MENU
-			"Pine" EXEC ULUB_TERM -T "Program pocztowy Pine" -e pine 
-			"Elm"  EXEC ULUB_TERM -T "Program pocztowy Elm" -e elm
-			"Xmh"  EXEC xmh
-		"Programy pocztowe" END
-		"Emulator terminala" MENU
-			"Minicom" EXEC xminicom
-			"Seyon"   EXEC seyon
-		"Emulator terminala" END
-		"Telnet"     EXEC ULUB_TERM -e telnet %a(Telnet,Podaj nazwe hosta:)
-		"Ssh"        EXEC ULUB_TERM -e ssh %a(Ssh,Podaj nazwe hosta:)
-		"Ftp"        EXEC ULUB_TERM -e ftp %a(Ftp,Podaj nazwe hosta:)
-		"Irc"        EXEC ULUB_TERM -e irc %a(Irc,Podaj swoj pseudonim:)
-		"Ping"       EXEC ULUB_TERM -e ping %a(Ping,Podaj nazwe hosta:)
-		"Talk"       EXEC ULUB_TERM -e talk %a(Talk,Podaj nazwe uzytkownika, z ktorym chcesz nawiazac polaczenie:)
-	"Internet" END
-
-	"Editory" MENU
-		"XFte"    EXEC xfte
-		"XEmacs"  EXEC xemacs || emacs
-		"XJed"    EXEC xjed 
-		"NEdit"   EXEC nedit
-		"Xedit"   EXEC xedit
-		"Editres" EXEC editres
-		"VI"      EXEC ULUB_TERM -e vi
-	"Editory" END
-	
-	"Dźwięk" MENU
-		"CDPlay"  EXEC workbone
-		"Xmcd"    EXEC xmcd 2> /dev/null
-		"Xplaycd" EXEC xplaycd
-		"Xmixer"  EXEC xmixer
-	"Dźwięk" END
-	
-    "Gry" MENU
-    	"Maze"      EXEC maze
-    	"Karty "    EXEC spider
-    	"Londownik" EXEC xlander
-    	"Szachy "   EXEC xboard
-    	"Xeyes"     EXEC xeyes -geometry 51x23
-    	"Xmahjongg" EXEC xmahjongg
-    	"Xlogo"     EXEC xlogo
-    	"Xroach"    EXEC xroach
-    	"Xtetris"   EXEC xtetris -color
-    	"Xvier"     EXEC xvier
-    	"Xgas"      EXEC xgas
-    	"Xkobo"     EXEC xkobo
-    	"xboing"    EXEC xboing -sound
-    	"XBill"     EXEC xbill
-    "Gry" END
+        "XTerm" EXEC xterm
+        "ATerm"  EXEC aterm
+	"Pulpity" WORKSPACE_MENU
 	
-	"Użytki" MENU
-		"Kalkulator"          EXEC xcalc
-		"Zegarek"             EXEC xclock
-		"Opcje Okna"          EXEC xprop | xmessage -center -title 'xprop' -file -
-		"Przeglądarka Fontów" EXEC xfontsel
-		"Szkło Powiększające" EXEC xmag
-		"Mapa Kolorów"        EXEC xcmap
-		"XKill"               EXEC xkill
-		"Clipboard"           EXEC xclipboard
-	"Użytki" END
+	"Aplikacje" OPEN_MENU | wmconfig --output wmaker --rootmenu Applications 2>/dev/null
 
-	"Selekcyjne" MENU
-		"Kopia"                  EXEC echo '%s' | wxcopy
-		"Poczta do ..."          EXEC ULUB_TERM -name mail -T "Pine" -e pine %s
+	"Zaznaczenie" MENU
+		"Kopiuj"                  SHEXEC echo '%s' | wxcopy
+		"Poczta do ..."          EXEC xterm -name mail -T "Pine" -e pine %s
 		"Serfuj do ..."          EXEC netscape %s
-		"Pobierz Manual ..."     EXEC MANUAL_SEARCH(%s)
-		"Połącz się z ..."       EXEC telnet %s
-		"Pobierz plik z FTP ..." EXEC ftp %s
-	"Selekcyjne" END
+		"Szukaj w manualach ..."     SHEXEC MANUAL_SEARCH(%s)
+	"Zaznaczenie" END
 
 	"Ekran" MENU
-		"Ukryj Pozostałe"         HIDE_OTHERS
+		"Ukryj pozostałe"         HIDE_OTHERS
 		"Pokaż wszystko"          SHOW_ALL
-		"Uporządkowanie icon"     ARRANGE_ICONS
-		"Odswież"                 REFRESH
+		"Uporządkuj ikony"        ARRANGE_ICONS
+		"Odśwież"                 REFRESH
 		"Zablokuj"                EXEC xlock -allowroot -usefirst
-		"Zachowaj Sesje"          SAVE_SESSION
-		"Wyczyść zachowaną sesje" CLEAR_SESSION
+		"Zachowaj sesję"          SAVE_SESSION
+		"Wyczyść zachowaną sesję" CLEAR_SESSION
 	"Ekran" END
 
 	"Wygląd" MENU
